pkg/config/middleware.go

```diff
@@ -0,0 +1,136 @@
+package config
```

Comment:
Let's separate this logic in 2 files.
- config/kubeClient.go
- rbac/authMiddleware.go

Note the different package names. I want to start a new package to keep everything related to rbac.